src/overlays/actors/ovl_En_Reeba/z_en_reeba.c

```diff
@@ -499,76 +510,76 @@ void func_80AE5C38(EnReeba* this, PlayState* play) {
                     }
                     FALLTHROUGH;
-                case 13: // hookshot/longshot
-                    if ((this->actor.colChkInfo.health > 2) && (this->unk_27E != 4)) {
-                        this->unk_27E = 4;
+                case LEEVER_DMGEFF_HOOKSHOT: 
+                    if ((this->actor.colChkInfo.health > 2) && (this->stunType != LEEVER_STUN_OTHER)) {
+                        this->stunType = LEEVER_STUN_OTHER;
                         Actor_SetColorFilter(&this->actor, 0, 0xFF, 0, 0x50);
                         Audio_PlayActorSfx2(&this->actor, NA_SE_EN_GOMA_JR_FREEZE);
-                        this->actionfunc = func_80AE58EC;
+                        this->actionfunc = EnReeba_SetupStunned;
                         break;
                     }
                     FALLTHROUGH;
-                case 14:
-                    this->unk_27C = 6;
+                case LEEVER_DMGEFF_OTHER:
+                    this->unkDamageField = 6;
                     Actor_ApplyDamage(&this->actor);
                     if (this->actor.colChkInfo.health == 0) {
                         Audio_PlayActorSfx2(&this->actor, NA_SE_EN_RIVA_DEAD);
                         Enemy_StartFinishingBlow(play, &this->actor);
-                        this->actionfunc = func_80AE5BC4;
+                        this->actionfunc = EnReeba_SetupDie;
                     } else {
-                        if (this->actionfunc == func_80AE5E48) {
+                        if (this->actionfunc == EnReeba_StunRecover) {
                             this->actor.shape.rot.x = this->actor.shape.rot.z = 0;
                         }
                         Audio_PlayActorSfx2(&this->actor, NA_SE_EN_RIVA_DAMAGE);
-                        this->actionfunc = func_80AE57F0;
+                        this->actionfunc = EnReeba_SetupDamaged;
                     }
                     break;
-                case 3: // ice arrows/ice magic
+                case LEEVER_DMGEFF_ICE:
                     Actor_ApplyDamage(&this->actor);
-                    this->unk_27C = 2;
-                    this->unk_27E = 2;
+                    this->unkDamageField = 2;
+                    this->stunType = LEEVER_STUN_ICE;
                     Actor_SetColorFilter(&this->actor, 0, 0xFF, 0, 80);
-                    this->actionfunc = func_80AE58EC;
+                    this->actionfunc = EnReeba_SetupStunned;
                     break;
-                case 1: // unknown
-                    if (this->unk_27E != 4) {
-                        this->unk_27E = 4;
+                case LEEVER_DMGEFF_UNK:
+                    if (this->stunType != LEEVER_STUN_OTHER) {
+                        this->stunType = LEEVER_STUN_OTHER;
                         Actor_SetColorFilter(&this->actor, 0, 0xFF, 0, 80);
-                        this->actionfunc = func_80AE58EC;
+                        this->actionfunc = EnReeba_SetupStunned;
                     }
                     break;
```

Comment:
Can this switch case fit a
```c
default:
    break;
```
at the end?

Reply:
Right, we made a somewhat recent style decision for switches, add a default case if it matches, and put line breaks in between cases
Like this one here for example
https://github.com/zeldaret/oot/blob/b57f2162ee71b23643fd4e4133f7455905c2969a/src/code/z_kankyo.c#L351-L383

Reply:
bump on adding a space after the `break;` here